src/components/NotFoundMessage.tsx

```diff
@@ -31,19 +32,27 @@ const NotFoundMessage: React.FunctionComponent<NotFoundMessageProps> = ({
   ...props
 }) => {
   return (
-    <EmptyState variant={EmptyStateVariant.large} {...props}>
-      <EmptyStateIcon icon={ExclamationCircleIcon} color={dangerColor.value} />
-      <Title headingLevel="h4" size="lg">
-        {errorTitle}
-      </Title>
+    <EmptyState variant={EmptyStateVariant.lg} {...props}>
+      <EmptyStateHeader
+        titleText={<>{errorTitle}</>}
```

Comment:
Why do we need the React Fragment ( `<>...</>` ) tags here?

Reply:
oh im not sure, i would guess the codemods threw that in there